addons/account/__init__.py

```diff
@@ -7,13 +7,14 @@
 import report
 
 from odoo import api, SUPERUSER_ID
+from openerp.tools import config
 
 
 def _auto_install_l10n(cr, registry):
     #check the country of the main company (only) and eventually load some module needed in that country
     env = api.Environment(cr, SUPERUSER_ID, {})
     country_code = env.user.company_id.country_id.code
-    if country_code:
+    if country_code and not config.options['test_enable']:
```

Comment:
This won't fix the production environment case.
@qdp-odoo What is the best way to skip this hook in our environmnets?